### PR TITLE
success and error toast functions added

### LINKED ISSUE #66: [Impl] Create toast for app
Create functions for display error and success messages using toastify. You can use below library.
https://www.npmjs.com/package/react-toastify

Create separate file for toastify and write 2 functions for display error message and success message.

### PR DESCRIPTION
**Pull request by niruthikka24**

### CODE REVIEW

File: src/components/toast.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { toast } from "react-toastify";
+
```
Comment (thilinawaks1999) [inline src/components/toast.js:2]: Can we set the position to top-right

Comment (niruthikka24) [reply src/components/toast.js:2]: When the position is top-right, it's hiding the logout button. Is that okay?

Comment (thilinawaks1999) [reply src/components/toast.js:2]: Display in middle is not suitable I think. I think it is no matter coz toast will disappear after few seconds. If it is a issue we can set position to bottom-right, right ?

Comment (niruthikka24) [reply src/components/toast.js:2]: Okay, then will set it to top-right itself, because as you said it'll disappear after a few seconds.

**Review by thilinawaks1999 — APPROVED** [review]

**Pull request merged**